blackbox/docs/sql/ddl/replication.txt

```diff
@@ -33,10 +66,18 @@ some examples.
 ===== =========================================================================
 Range Explanation
 ===== =========================================================================
-0-1   Will create 0 or 1 replicas depending on the number of available nodes
+0-1   Will create no replicas if only one node. This will result in a yellow
+      cluster health.
+
+      One replica if more than one node.
 ----- -------------------------------------------------------------------------
-2-4   Table requires at least 2 replicas to be fully replicated. Will create up
-      to 4 if nodes are added.
+2-4   Table requires at least two replicas to be fully replicated.
+
+      Will create up to four replicas if nodes are added.
+
+      If you have less than three nodes, one or more replica shards will be
+      located on the same node as the primary shard. This will result in a
+      yellow cluster health.
 ----- -------------------------------------------------------------------------
```

Comment:
ES has a "restriction" edge-case here. For replication purpose Crate uses the ES API.

* If a table with `2-4` range will be created **from scratch** on less than three nodes it will only create primary shards without any replication
* If the table schema gets changed using `alter table` to set `2-4` on a cluster with less than three nodes, it will stay and keep the number of replicas (and therefore shards) it had before.

Reply:
@mikethebeer could you provide proposed replacement text for this whole section? I am not sure I understand your meaning